ldap/servers/plugins/acl/acllist.c

```diff
@@ -129,7 +129,7 @@ acl_be_state_change_fnc(void *handle __attribute__((unused)), char *be_name, int
          * Just get the first suffix--if there are multiple XXX ?
         */
 
-        if ((sdn = slapi_be_getsuffix(be)) == NULL) {
+        if ((sdn = slapi_be_getsuffix(be, 0)) == NULL) {
```

Comment:
I think there have been some changes around this, @mreynolds389 may know more. Either way, I don't think this change should be in this PR, I think that you may need to rebase to the latest commits.